alexa-briefing.php

```diff
@@ -4,24 +4,62 @@
 
 class Alexa_Briefing {
 	public function briefing_request() {
+		$responses = [];
 
-		$briefing = get_posts( [
-			'posts_per_page' => 1,
-			'post_type' => 'alexawp-briefing',
-			'post_status' => 'publish',
-		] );
-
-		if ( $briefing && ! empty( $briefing[0] ) ) {
+		// This logic could be both abstracted and used with array_map().
+		foreach (
+			get_posts( [
+				'meta_query' => [
```

Comment:
With the logic below as it is currently, this meta value is required for the briefing to contain any content.